### ISSUE TITLE
[FEAT]: Check Shulker Boxes in Item Type Currencies

### ISSUE THREAD

**Issue by KarlosWGeek** [TheNewEconomy/TNE-Bukkit#506]
### Feature Description

In a item-based economy (item = currency), the plugin should check inside Shulker Boxes just like it already does with Ender Chests.

### Reasoning

Just like Ender Chests, Shulker Boxes are used to carry large quantities of items. So, just like the plugin checks Ender Chests for the item-currency and shows it on the player's balance, the same should be applied to Shulker Boxes.

**Comment by NickAwrist** [comment]
This would be really cool tbh

**Comment by creatorfromhell** [comment]
> This would be really cool tbh

Indeed, I've added this to the todo for while I'm revamping the currency stuff.
